lib/internal/process.js

```diff
@@ -84,8 +84,7 @@ function setup_hrtime() {
         const needsBorrow = nsec < 0;
         return [needsBorrow ? sec - 1 : sec, needsBorrow ? nsec + 1e9 : nsec];
       }
-      throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
-                                 'process.hrtime()', 'Array');
+      throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'time', 'Array');
```

Comment:
This could be improved by rewriting the statements to something like:

```js
if (!Array.isArray(time)) {
  throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'time', 'Array', time);
}
if (time.length !== 2) {
  throw new errors.TypeError('ERR_TYPE_X', 'time', 2, time);
}
const sec = (hrValues[0] * 0x100000000 + hrValues[1]) - time[0];
const nsec = hrValues[2] - time[1];
const needsBorrow = nsec < 0;
return [needsBorrow ? sec - 1 : sec, needsBorrow ? nsec + 1e9 : nsec];
```